### PR TITLE
Add messages to remove user

### PR DESCRIPTION
**Pull request by deads2k**
Fixes https://github.com/openshift/origin/issues/2785 (messages)

@liggitt

### CODE REVIEW

**Comment by liggitt** [comment]
Figure out with @smarterclayton whether we want cmdutil.Out or not... he's axing it in #2835

File: pkg/cmd/admin/policy/remove_from_project.go [hunk 1]
```diff
@@ -109,8 +121,24 @@ func (o *RemoveFromProjectOptions) Run() error {
 			if err != nil {
 				return err
 			}
+
+			if diff := existingGroups.Difference(currBinding.Groups); len(diff) != 0 {
+				fmt.Fprintf(o.Out(), "Removing %s/%s from groups %v in project %s.\n", currBinding.RoleRef.Namespace, currBinding.RoleRef.Name, diff.List(), o.BindingNamespace)
```
Comment (liggitt) [inline pkg/cmd/admin/policy/remove_from_project.go]: for cluster roles: `Removing admin role from...`
for namespaced roles: `Removing ns/thinger role from...`

File: pkg/cmd/admin/policy/remove_from_project.go [hunk 2]
```diff
@@ -109,8 +121,24 @@ func (o *RemoveFromProjectOptions) Run() error {
...
+			}
+			if diff := existingUsers.Difference(currBinding.Users); len(diff) != 0 {
+				fmt.Fprintf(o.Out(), "Removing %s/%s from users %v in project %s.\n", currBinding.RoleRef.Namespace, currBinding.RoleRef.Name, diff.List(), o.BindingNamespace)
+				usersRemoved.Insert(diff.List()...)
+			}
 		}
 	}
 
+	if diff := util.NewStringSet(o.Groups...).Difference(groupsRemoved); len(diff) != 0 {
+		fmt.Fprintf(o.Out(), "Groups %v did not have access to project %s.\n", diff.List(), o.BindingNamespace)
```
Comment (liggitt) [inline pkg/cmd/admin/policy/remove_from_project.go]: Be careful how you word that... if they have cluster bindings, they do have access. Maybe something like "Groups %v were not bound to roles in project %s.\n"?

**Comment by smarterclayton** [comment]
Do not want.  Big kids initialize their structs up front.

**Comment by deads2k** [comment]
> Do not want.  Big kids initialize their structs up front.

There is some merit to avoiding a panic if the default struct is used.

**Comment by deads2k** [comment]
comments addressed.

**Comment by smarterclayton** [comment]
It's called testing.

**Comment by liggitt** [comment]
LGTM

**Comment by deads2k** [comment]
[merge]

**Comment by deads2k** [comment]
[merge]

**Pull request merged**